smithy-typescript-codegen/src/main/resources/software/amazon/smithy/typescript/codegen/integration/default_readme_client.md.template

```diff
@@ -0,0 +1,136 @@
+<!-- generated file, do not edit directly -->
+
+# ${packageName}
+
+## Description
+
+SDK for JavaScript ${serviceId} Client for Node.js, Browser and React Native.
+
+${documentation}
+
+## Installing
+To install the this package, simply type add or install ${packageName}
+using your favorite package manager:
+- `npm install ${packageName}`
+- `yarn add ${packageName}`
+- `pnpm add ${packageName}`
+
+## Getting Started
+
+### Import
+
+To send a request, you only need to import the `${serviceId}Client` and
+the commands you need, for example `${commandName}Command`:
+
+```js
+// ES5 example
+const { ${serviceId}Client, ${commandName}Command } = require("${packageName}");
+```
+
+```ts
+// ES6+ example
+import { ${serviceId}Client, ${commandName}Command } from "${packageName}";
+```
+
+### Usage
+
+To send a request, you:
+
+- Initiate client with configuration (e.g. credentials, region).
```

Comment:
maybe remove region from the example if it's not a core part of the generic clients